### PR TITLE
docs: add equation for linear lr

### LINKED ISSUE #68058: Some lr-schedulers docs seems to have typos/missing information
## 📚 Documentation

According to [ConstantLR](https://pytorch.org/docs/stable/generated/torch.optim.lr_scheduler.ConstantLR.html#torch.optim.lr_scheduler.ConstantLR) "*Decays the learning rate of each parameter group by a small constant factor until the number of epoch reaches a pre-defined milestone: total_iters*"  i.e with `factor=0.5`, `lr=0.05`  and `total_iters=4` I would assume

```
#lr = 0.025 
#lr = 0.0125
#lr = 0.00625
#lr = 0.003125
#lr = 0.5  #Back to initial
```
i.e

```
lr[i+1] = lr[0]*start_factor**i
```
where the docs writes 

```
 # Assuming optimizer uses lr = 0.05 for all groups
# lr = 0.025   if epoch == 0
# lr = 0.025   if epoch == 1
# lr = 0.025   if epoch == 2
# lr = 0.025   if epoch == 3
# lr = 0.05    if epoch >= 4
```

In the  [LinearLR](https://pytorch.org/docs/stable/generated/torch.optim.lr_scheduler.LinearLR.html) the learning-rate examples

```
# Assuming optimizer uses lr = 0.05 for all groups
# lr = 0.025    if epoch == 0
# lr = 0.03125  if epoch == 1
# lr = 0.0375   if epoch == 2
# lr = 0.04375  if epoch == 3
# lr = 0.005    if epoch >= 4
 scheduler = LinearLR(self.opt, start_factor=0.5, total_iters=4)
```

increases after epoch 0 until epoch 4 where it drops again (why?). As far as I understand LinearLR we get the learning-rate in each step by

```
lr[i+1] = lr[i]*start_factor**(i)
```

Generally it would be great, if you provided a (short) formular of how you calculate each learning rate in the schedulers since the wording can be interpreted differently by different persons but formulars (often) does not.

### PR DESCRIPTION
**Pull request by jaketae**
Fixes https://github.com/pytorch/pytorch/issues/68058.

### CODE REVIEW

File: torch/optim/lr_scheduler.py [hunk 1]
```diff
@@ -487,10 +487,16 @@ def _get_closed_form_lr(self):
 
 
 class LinearLR(_LRScheduler):
-    """Decays the learning rate of each parameter group by linearly changing small
-    multiplicative factor until the number of epoch reaches a pre-defined milestone: total_iters.
-    Notice that such decay can happen simultaneously with other changes to the learning rate
-    from outside this scheduler. When last_epoch=-1, sets initial lr as lr.
+    r"""Decays the learning rate of each parameter group by linearly changing small
+    multiplicative factor until the number of epoch reaches a pre-defined milestone: total_iters,
+    or :math:`T_\text{max}`. :math:`K_\text{start}` and :math:`K_\text{end}`denote the start
```
Comment (albanD) [inline torch/optim/lr_scheduler.py:492]: I think you're missing a space that breaks the rendering: https://docs-preview.pytorch.org/71231/generated/torch.optim.lr_scheduler.LinearLR.html?highlight=linearlr#torch.optim.lr_scheduler.LinearLR

You can see this rendered doc in the "  Preview docs built from this PR" link in the bot comment on the PR.

Comment (jaketae) [inline torch/optim/lr_scheduler.py:492]:
```suggestion
    or :math:`T_\text{max}`. :math:`K_\text{start}` and :math:`K_\text{end}` denote the start
```

Comment (jaketae) [reply torch/optim/lr_scheduler.py]: Thanks for the heads up! Pushed the fix, and seems like things are rendering correctly now ([link](https://docs-preview.pytorch.org/71231/generated/torch.optim.lr_scheduler.LinearLR.html?highlight=linearlr#torch.optim.lr_scheduler.LinearLR)). I'm assuming the `numba` import errors are unrelated to this PR?

Comment (albanD) [reply torch/optim/lr_scheduler.py]: The eta_0 part still does not render properly right?

Comment (albanD) [reply torch/optim/lr_scheduler.py]: Yes CI is not in a good shape today: https://hud.pytorch.org/ci/pytorch/pytorch/master you can ignore

Comment (jaketae) [reply torch/optim/lr_scheduler.py]: > The eta_0 part still does not render properly right?

Is it not? It does look a bit awkward, but I assumed that's the way subscripts are rendered. Am I missing something here?

```
When last_epoch=-1, sets initial
     lr :math:`\eta_0` as lr.

.. math::
         \eta_t = \eta_0 \left(K_\text{start} + \frac{T_\text{curr}(K_\text{end} -
```

Thanks in advance!

**Comment by albanD** [comment]
The docs preview seem to be still bad. Did you build the doc locally to check this by any chance?
Otherwise, could you rebase on top of master and push again to re-trigger a clean CI?

**Comment by jaketae** [comment]
Accidentally merged during rebase! I'll close this in favor of a new PR.
